Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/Visuals/ProximityEffect/IProximityEffectObjectProvider.cs

```diff
@@ -36,5 +42,13 @@ public interface IProximityEffectObjectProvider
         /// This method allows iterating over the proximity scaled visuals. It should return the gameobject the scaling should be applied to.
         /// </summary>
         void ForEachProximityObject(Action<Transform> action);
+
+        /// <summary>
+        /// Allow for accessing / subscribing to the changed event for objects that show a proximity effect
+        /// </summary>
+        ProximityObjectsChangedEvent ProximityObjectsChanged
+        {
+            get;
+        }
```

Comment:
nit to match the formatting of the rest of the file

```suggestion
        ProximityObjectsChangedEvent ProximityObjectsChanged { get; }
```